### DISCUSSION TITLE
Bots & PvE Enthusiast, where to get started to make my own mods?

### DISCUSSION THREAD

**Discussion by ghost** [Joe91/fun-bots#338] [Q&A]
Hi there, and this one goes for anybody who reads this message. How to get started into making a bots mod for a Battlefield/Frostbite game? I've searched for tutorials but.. Internet has very few to say about the topic.. only tutorial I saw was one for a "Plants Vs Zombies" game lol.. I have plenty of spare time and wanna learn how to do it. Battlefield Hardline, Battlefield 4, Battlefield 1 or V, I know theyre different beasts but.. still curious and wanting to make a mod for any of those.

**Comment by koopa990** [comment]
Have you considered joining the VU discord? They can help.
